GlucoseOnFhir/GlucoseMeterViewController.swift

```diff
@@ -288,12 +414,18 @@ class GlucoseMeterViewController: UITableViewController, GlucoseProtocol, FHIRPr
         patient.address = [patientAddress]
         patient.birthDate = patientBirthDate
         
-        print(patient.asJSON())
-        
-        fhir.createPatient(patient: patient)
+        FHIR.sharedInstance.createPatient(patient: patient) { (patient, error) -> Void in
```

Comment:
could be simplified to: `FHIR.sharedInstance.createPatient(patient: patient) { patient, error in`